### PR TITLE
Update operatingsystems-linux-nrpe3.md

### PR DESCRIPTION
**Pull request by SavCent**
## Description

Format to TabItem and add missing command available in plugin.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

**Review by omercier — APPROVED** [review]

**Comment by omercier** [comment]
Thank you @SavCent!

File: pp/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md [hunk 1]
```diff
@@ -28,26 +28,38 @@ This Monitoring Connector relies on 3 components that are listed below:
...
-* Process-Generic : Check linux processes
-* Systemd-Sc-Status : Check services managed by *systemd*
-* Traffic-Generic-Name : Check Traffic (single interface)
-* Traffic-Global : Check Traffic (all interfaces or filtered with regexp)
+<Tabs groupId="sync">
+<TabItem value="OS-Linux-NRPE3-custom" label="OS-Linux-NRPE3-custom">
+
+| Service Alias       | Service Template                          | Service Description                           |
+|:--------------------|:------------------------------------------|:----------------------------------------------|
+| Check-Plugin *      | OS-Linux-Check-Plugin-NRPE3-custom        | SSH execution commands in a remote host       |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md:38]:
```suggestion
| Check-Plugin        | OS-Linux-Check-Plugin-NRPE3-custom        | SSH execution commands in a remote host       |
```

File: pp/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md [hunk 2]
```diff
@@ -28,26 +28,38 @@ This Monitoring Connector relies on 3 components that are listed below:
...
+| File-Date-Generic   | OS-Linux-File-Date-Generic-NRPE3-custom   | Check time of files/directories               |
+| File-Size-Generic   | OS-Linux-File-Size-Generic-NRPE3-custom   | Check size of files/directories               |
+| Inodes              | OS-Linux-Inodes-NRPE3-custom              | Check Inodes space usage                      |
+| Is-File-Generic     | OS-Linux-Is-File-Generic-NRPE3-custom     | Check the presence of a file                  |
+| Is-Not-File-Generic | OS-Linux-Is-Not-File-Generic-NRPE3-custom | Check the absence of a file                   |
+| Load                | OS-Linux-Load-NRPE3-custom                | Check system load-average                     |
+| Ntp                 | OS-Linux-Ntp-NRPE3-custom                 | Check ntp daemons                             |
+| OpenFile *          | OS-Linux-OpenFile-NRPE3-custom            | Check open files                              |
+| Packet-Errors       | OS-Linux-Packet-Errors-NRPE3-custom       | Check packet errors and discards on interfaces|
+|	Pending-Updates *   | OS-Linux-Pending-Updates-NRPE3-custom     | Check pending updates                         |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md:53]:
```suggestion
| Pending-Updates        | OS-Linux-Pending-Updates-NRPE3-custom     | Check pending updates                         |
```

File: pp/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md [hunk 3]
```diff
@@ -28,26 +28,38 @@ This Monitoring Connector relies on 3 components that are listed below:
...
+| Process-Generic     | OS-Linux-Process-Generic-NRPE3-custom     | Check linux processes                         |
+| Systemd-Journal *   | OS-Linux-Systemd-Journal-NRPE3-custom     | Count journal entries                         |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md:55]:
```suggestion
| Systemd-Journal   | OS-Linux-Systemd-Journal-NRPE3-custom     | Count journal entries                         |
```

**Review by cg-tw — APPROVED** [review]

File: pp/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md [hunk 4]
```diff
@@ -29,25 +29,36 @@ This Monitoring Connector relies on 3 components that are listed below:
...
+| Pending-Updates        | OS-Linux-Pending-Updates-NRPE3-custom     | Check pending updates                         |
+| Process-Generic     | OS-Linux-Process-Generic-NRPE3-custom     | Check linux processes                         |
+| Systemd-Journal   | OS-Linux-Systemd-Journal-NRPE3-custom     | Count journal entries                         |
+| Systemd-Sc-Status   | OS-Linux-Systemd-Sc-Status-NRPE3-custom   | Check services managed by systemd             |
+| Traffic             | OS-Linux-Traffic-NRPE3-custom             | Check Traffic                                 |
+
+
+> The services listed above are created automatically when the **OS-Linux-NRPE3-custom** host template is used.
+
+</TabItem>
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md:61]:
```suggestion
</TabItem>
</Tabs>
```

**Pull request merged**